docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -4142,6 +4142,17 @@ kafka:
   # CLI flag: -ingest-storage.kafka.wait-strong-read-consistency-timeout
   [wait_strong_read_consistency_timeout: <duration> | default = 20s]
 
+  # (experimental) The record version preferred by this producer.
+  # CLI flag: -ingest-storage.kafka.producer-supported-record-version
+  [producer_supported_record_version: <int> | default = 0]
+
+  # (experimental) The highest supported record version understood by this
+  # consumer. Versions lower than this value are also assumed to be supported.
```

Comment:
```suggestion
  # (experimental) The highest supported record version that this
  # consumer understands. Versions lower than this value are assumed to also be supported.
```

Reply:
I dropped this config option entirely, so this sentence doesn't exist anymore :+1: